src/ohri-form.component.tsx

```diff
@@ -28,6 +28,7 @@ import LoadingIcon from './components/loaders/loading.component';
 import OHRIFormSidebar from './components/sidebar/ohri-form-sidebar.component';
 import WarningModal from './components/warning-modal.component';
 import styles from './ohri-form.component.scss';
+import { isPostSubmissionEnabled } from './utils/program-enrolment-helper';
```

Comment:
Why the `isPostSubmissionEnabled` helper part of the Program Enrolment Helpers? My recommendation is to put this helper on a more generic place  / file.

Reply:
Done